tests/python/unittest/test_gluon_rnn.py

```diff
@@ -502,7 +498,6 @@ def test_cell_fill_shape():
 @assert_raises_cudnn_disabled()
 def test_layer_fill_shape():
     layer = gluon.rnn.LSTM(10)
-    layer.hybridize()
```

Comment:
Removing this hybridize call should be enough. is there any reason we are removing all hybridize calls ?

Reply:
The same error on test_norm still occurs if I only remove this one call, so I was trying out all possible cases